index.php

```diff
@@ -19,7 +19,7 @@
     
         <div class="section section-announcement">
           <div class='wrapper'>
-            <p><a href='/12/'>New: Keyman 12.0 is now available!</a></p>
+            <p style='font-size:2em'><a href='/training/'>🎓 New! Keyman training in Switzerland, Feb 2020</a></p>
```

Comment:
In a medium viewport (between ~846px to ~1080px wide), the title is getting covered up
![image](https://user-images.githubusercontent.com/7358010/69019356-d9d38b80-09e2-11ea-9e0a-71ba3c1a1ef4.png)